xml/System.Collections.Generic/List`1.xml

```diff
@@ -93,6 +93,9 @@
   
  For an immutable version of the <xref:System.Collections.Generic.List%601> class, see <xref:System.Collections.Immutable.ImmutableList%601>.  
   
+> [!NOTE]
```

Comment:
This note needs to be added inside the member tag where we're adding the interactive sample. So probably on line 845 (if you look at the old file without this addition).
Also, use the include file for this instead of straight text like the following

```markdown
[!INCLUDE[interactive-note](~/includes/csharp-interactive-note-some.md)]
```

Reply:
Done, thanks Maira.

Reply:
@mairaw -- Maira, thank you for the explanations.  I changed the note to be an include and moved it. 

I now see there are several of these -- I'm creating a combined PR for all five:

https://docs.microsoft.com/en-us/dotnet/api/system.collections.generic.list-1
https://docs.microsoft.com/en-us/dotnet/api/system.collections.generic.list-1.add
https://docs.microsoft.com/en-us/dotnet/api/system.collections.generic.list-1.find
https://docs.microsoft.com/en-us/dotnet/api/system.collections.generic.list-1.remove
https://docs.microsoft.com/en-us/dotnet/api/system.collections.generic.list-1.sort

Reply:
Done.